### PR TITLE
Updated branch based on recommended PR changes

### PR DESCRIPTION
**Pull request by bfotheri**

### CODE REVIEW

File: submitted_models/marble_husky_sensor_config_1/launch/spawner.rb [hunk 1]
```diff
@@ -20,6 +20,10 @@ def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
         <wheel_separation>#{0.45649 * 1.5}</wheel_separation>
         <wheel_radius>0.1651</wheel_radius>
         <topic>/model/#{_name}/cmd_vel_relay</topic>
+        <min_velocity>-1</min_velocity>
+        <max_velocity>1</max_velocity>
+        <min_acceleration>-2</min_acceleration>
```
Comment (caguero) [inline submitted_models/marble_husky_sensor_config_1/launch/spawner.rb:25]: This was a typo, it should be `-3`.

File: submitted_models/robotika_x2_sensor_config_1/launch/spawner.rb [hunk 2]
```diff
@@ -19,6 +19,10 @@ def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
   "      <wheel_separation>#{0.33559 * 1.23}</wheel_separation>\n"\
   "      <wheel_radius>0.098</wheel_radius>\n"\
   "      <topic>/model/#{_name}/cmd_vel_relay</topic>\n"\
+  "      <min_velocity>-1</min_velocity>\n"\
```
Comment (caguero) [inline submitted_models/robotika_x2_sensor_config_1/launch/spawner.rb:22]: Just checking, is this change from a `robotika` model on purpose?

Comment (bfotheri) [reply submitted_models/robotika_x2_sensor_config_1/launch/spawner.rb:22]: No, it's not. We can ignore this. Sorry about that.

Comment (bfotheri) [reply submitted_models/robotika_x2_sensor_config_1/launch/spawner.rb:22]: I thought this was requested inside the model_pr_426-> master pull request? Perhaps it was by mistake?
robotica_x2

Comment (caguero) [reply submitted_models/robotika_x2_sensor_config_1/launch/spawner.rb:22]: Yes, sorry for the confusion, I copied this comment from a different team.

**Pull request merged**